examples/gatsby/gatsby-spa/.env.local.example

```diff
@@ -0,0 +1,6 @@
+# Rename this file to .env.local and fill in the values for the following environment variables
```

Comment:
I had to name my file .env.development in order to get it to pick up the env vars, and that seems to be what Gatsby suggests when running the 'npm run develop' command. Curious if other found the same, and if so, we should update this filename and instructions to use .env.development instead.